### ISSUE TITLE
"--from-stdin" not available

### ISSUE THREAD

**Issue by bmiklautz** [FreeRDP/FreeRDP#1004]
In master it's not possible to read credentials from stdin this functionality seems to got lost during command line change.

**Comment by bmiklautz** [comment]
@jflorian the syntax style is one thing but I wouldn't call it hurried. The feature itself isn't really lost (see #1466) but got a little bit more complex ;(.
But it will be available again.

**Comment by jflorian** [comment]
@bmiklautz Sorry for the negativity, but I was frustrated with the change.  The run-time help stated that the (old) long options were still available when clearly they aren't.  I found the PR and discussion that went along with it.  I'll admit that the feature does seem to be going in the right direction.  I still don't like the slash-style options -- feels too Windowsy, but hey that's the perogitive of the authors.  (Ideally, all cross-platform apps would "when in Rome, do like the Romans" but I don't doubt there's plenty of bigger fish to fry.)

In the end, I discovered that I could leave the --from-stdin option off (to make the syntax legal) and xfreerdp would still pull the credentials from there anyway.  So I'm happy enough for now.
